client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
 				<button
 					className="conversation-caterpillar__count"
 					onClick={ this.handleTickle }
-					title={
-						commentCount > 1
-							? translate( 'View comments from %(commenterName)s and %(count)d more', {
-									args: {
-										commenterName: lastAuthorName,
-										count: commentCount - 1,
-									},
-								} )
-							: translate( 'View comment from %(commenterName)s', {
-									args: {
-										commenterName: lastAuthorName,
-									},
-								} )
-					}
+					title={ translate(
+						'View %(count)s more comments for this post',
+						'View %(count)s more comment for this post',
+						{
+							count: +commentCount,
+							args: {
+								count: commentCount,
+							},
+						}
+					) }
 				>
-					{ commentCount > 1
-						? translate( '%(commenterName)s and %(count)d more', {
-								args: {
-									commenterName: lastAuthorName,
-									count: commentCount - 1,
-								},
-							} )
-						: translate( '%(commenterName)s commented', {
-								args: {
-									commenterName: lastAuthorName,
-								},
-							} ) }
+					{ commentCount > 1 &&
+						uniqueAuthorsCount > 1 &&
+						translate( '%(count)d more comments from %(commenterName)s and others', {
+							args: {
+								commenterName: lastAuthorName,
+								count: commentCount,
+							},
+						} ) }
+					{ commentCount > 1 &&
+						uniqueAuthorsCount === 1 &&
+						translate( '%(count)d more comments from %(commenterName)s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comments from %(commenterName)s and %(count)d more' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).